admin/src/components/DeletedUserFormular.vue

```diff
@@ -4,19 +4,36 @@
       {{ $t('removeNotSelf') }}
     </div>
     <div v-else class="mt-5">
-      <b-form-checkbox switch size="lg" v-model="checked">
-        <div>{{ item.deletedAt ? $t('undelete_user') : $t('delete_user') }}</div>
-      </b-form-checkbox>
-
       <div class="mt-3 mb-5">
-        <b-button v-if="checked && item.deletedAt === null" variant="danger" @click="deleteUser">
+        <b-button
+          v-if="item.deletedAt === null"
+          variant="danger"
+          v-b-modal.delete-user-modal
+          @click="showModal('deleteUser')"
```

Comment:
What you use `v-b-modal` for when you reference the modal by a parameter in `showModal('deleteUser')` ?
Am I missing something?

Reference: https://stackoverflow.com/questions/49123108/vuejs-how-to-use-v-b-modal-directive-in-bootstrapvue

```
 methods: {
    showModal() {
      this.$refs.myModalRef.show();
    },
  }
  ```

Reply:
Resolved in https://github.com/gradido/gradido/pull/2880/commits/17f7e8adf0ce2928c5b4f3bb9a1c3c13c69066a7